scripts/run_bare_execution_tests.py

```diff
@@ -32,55 +34,44 @@
 import click
 
 # Where we expect to find the example files.
-E2E_DIR = "e2e/scripts"
+E2E_DIR = "e2e_playwright"
 
 EXCLUDED_FILENAMES: Set[str] = set()
 
-# st_experimental_rerun.py calls st.experimental_rerun which raises a
-# RerunException when called within plain Python.
-EXCLUDED_FILENAMES.add("st_experimental_rerun.py")
-
 # Since there is not DISPLAY set (and since Streamlit is not actually running
 # and fixing Matplotlib in these tests), we set the MPL backend to something
 # that doesn't require a display.
 os.environ["MPLBACKEND"] = "Agg"
 
 
 def _command_to_string(command):
-    if isinstance(command, list):
-        return " ".join(command)
-    else:
-        return command
+    return " ".join(command) if isinstance(command, list) else command
 
 
-def _get_filenames(dir):
-    dir = os.path.abspath(dir)
+def _get_filenames(folder):
+    folder = os.path.abspath(folder)
```

Comment:
nit: it's probably not a good idea to re-use a parameter variable

Reply:
Changed 👍